LLama/runtimes/build/LLamaSharp.Backend.OpenCL.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package >
+  <metadata>
+    <id>LLamaSharp.Backend.OpenCL</id>
+    <version>$version$</version>
+    <title>LLamaSharp.Backend.OpenCL - OpenCL Backend for LLamaSharp</title>
+    <authors>llama.cpp Authors</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <icon>icon512.png</icon>
+    <projectUrl>https://github.com/SciSharp/LLamaSharp</projectUrl>
+    <description>LLamaSharp.Backend.OpenCL is a backend for LLamaSharp to use with OpenCL.</description>
+    <releaseNotes></releaseNotes>
+    <copyright>Copyright 2023 The llama.cpp Authors. All rights reserved.</copyright>
+    <tags>LLamaSharp LLama LLM GPT AI ChatBot SciSharp</tags>
+  </metadata>
+
+  <files>
+    <file src="LLamaSharpBackend.props" target="build/netstandard2.0/LLamaSharp.Backend.OpenCL.props" />
+    
+    <file src="runtimes/deps/clblast/llama.dll" target="runtimes\win-x64\native\clblast\llama.dll" />
+    <file src="runtimes/deps/clblast/libllama.so" target="runtimes\linux-x64\native\clblast\libllama.so" />
+    <file src="runtimes/deps/clblast/clblast.dll" target="runtimes\win-x64\native\clblast\clblast.dll" />
+    <file src="runtimes/deps/clblast/libclblast.so" target="runtimes\linux-x64\native\clblast\libclblast.so" />
```

Comment:
Should this be removed, since we've decided not to distribute lbclblast.so for now?

Reply:
yup, and there was a 2nd place it needed to be nixed from too.